<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>352</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Launcher Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="deliveryGroupBox">
     <property name="title">
      <string>Delivery settings:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLabel" name="bandwithLabel">
        <property name="text">
         <string>Bandwith limits:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="uploadSpin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> KiB/s</string>
        </property>
        <property name="minimum">
         <number>32</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="uploadCheckBox">
        <property name="text">
         <string>upload</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="downloadCheckBox">
        <property name="text">
         <string>download</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="downloadSpin">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> KiB/s</string>
        </property>
        <property name="minimum">
         <number>32</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="generalLabel">
        <property name="text">
         <string>General:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QCheckBox" name="seedCheckBox">
        <property name="text">
         <string>seed game to other players</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QCommandLinkButton" name="debugLinkButton">
        <property name="text">
         <string>Generate Debug data for support Team</string>
        </property>
        <property name="icon">
         <iconset resource="assets.qrc">
          <normaloff>:/images/attention.png</normaloff>:/images/attention.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="connectionsSpin">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="value">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="bandwithLabel_2">
        <property name="text">
         <string>max connections</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="supportLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Problems or Questions? Email us at &lt;a href=&quot;mailto:contact@spicyhorse.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;contact@spicyhorse.com&lt;/span&gt;&lt;/a&gt;&lt;br/&gt;Or visit our forums at: &lt;a href=&quot;http://spicyworld.spicyhorse.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;spicyworld.spicyhorse.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>downloadSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>uploadSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
